yamlExamples/1/example.yaml

```diff
@@ -0,0 +1,42 @@
+# same as defined in current version
+agent:
+  label: my_label
+  customWorkspace: path_to_workspace
+
+tools:
+  tool_1 : tool_1_name
+  tool_2 : tool_2_name
+
+# if true PR will be fetched and build will be started without merging it to the target branch
+# else PR will be merged to the target branch and then build will be started.
```

Comment:
Actually, I think that if a PR is opened then eventually it will be merged in the target branch and hence it does not make sense to build the PR without merging it in the target branch.